### PR TITLE
Remove possibility of setting future date

### PR DESCRIPTION
**Pull request by MartaKoprowska**
Also validate corectness of dates

### CODE REVIEW

**Comment by MartaKoprowska** [comment]
Resolves #200

File: frontend/src/views/DriveFormView.vue [hunk 1]
```diff
@@ -204,6 +205,7 @@ export default {
       searchText: '',
       confirmationOnline: false,
       confirmationOffline: false,
+      currentDate: new Date().toISOString().split('T')[0]
```
Comment (jacol12345) [inline frontend/src/views/DriveFormView.vue]: Please add missing comma (linter won't pass).

**Comment by jacol12345** [comment]
@MartaKoprowska Please add missing comma.

**Comment by adamgajzlerowicz** [comment]
Tested locally - test ok

**Pull request merged**